adal/src/main/java/com/microsoft/aad/adal/AuthenticationRequest.java

```diff
@@ -63,6 +63,8 @@
 
     private boolean mIsExtendedLifetimeEnabled = false;
 
+    private String mTelemetryRequestId;
```

Comment:
> String mTelemetryRequestId; [](start = 11, length = 28)

if correlationid is guaranteed to be the same per API request, we can reuse correlation id, right?